gears_cli/__main__.py

```diff
@@ -67,7 +68,29 @@ def print_res(res, res_id):
         pass
     print('%s\t%s' % (res_id, Colors.Bold(res)))
 
-@gears_cli.command()
+@gears_cli.command(help='Install give requirements')
+@click.option('--host', default='localhost', help='Redis host to connect to')
+@click.option('--port', default=6379, type=int, help='Redis port to connect to')
+@click.option('--password', default=None, help='Redis password')
+@click.option('--requirements-file', default=None, help='Path to requirements.txt file')
+@click.argument('requirements', nargs=-1, type=click.UNPROCESSED)
+def install_requirements(host, port, password, requirements_file, requirements):
+    r = create_connection(host, port, password);
+
+    requirements = [a for a in requirements]
```

Comment:
`requirements = list(requirements)` ?

Reply:
I think it will create list with one element that contains the requirements no?

Reply:
donno. `[...]` looks like a materialization of an iterator (it gets an iterator and returns a concrete list. but maybe i don't read this correctly.